src/content/projects_content.js

```diff
@@ -344,6 +344,34 @@ const projects = {
           contributors_url:
             'https://api.github.com/repos/anitab-org/stem-diverse-tv/contributors',
         },
+
+        {
+          image: {
+            source: require('./../assets/placeholder.png'),
+          },
+          title: 'Stem Diverse TV CMS',
+          description:
+            'CMS for managing content of STEM Diverse TV, an app which provides inspiring, motivating, supportive, educational and informative videos to promote and increase diversity in STEM.',
+          labels: [
+            { title: 'react js', color: '#BD6AAA' },
+            { title: 'javascript', color: '#F9D507' },
```

Comment:
@Rahulm2310 We are actually using `Typescript` in the `Stem Diverse TV CMS` project. So, IG it would be better to replace `Javascript` with `Typescript`.